pages/alerts.rst

```diff
@@ -8,9 +8,13 @@ Alerts
 
 Alerts are created using Event Definitions that consist of Conditions. When a given condition is met it will be stored as an Event and can be used to trigger a notification. If your system has an enterprise license, then Events may be combined to create Correlations.
 
-
 Graylog ships with default *alert conditions* and *alert notifications*, and both can be extended with :ref:`Plugins <plugins>`.
 
+.. toctree::
+   :titlesonly:
+
+   alerting/alerting-by-example.rst
+
```

Comment:
I would move this to the line before "if no events have been defined" - in the `Alerts & Events` section and use this:

```
.. toctree::
   :titlesonly:
   :hidden:

   alerting/alerting-by-example.rst
```

This will create a link on the right navigation but will not place that kind of lost link in the text itself.

What did you think?

Reply:
Like it. I did not know you could do that!

Done!